packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -834,6 +834,9 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     assert(() { _debugLocked = false; return true; }());
   }
 
+  /// The top-most [Route] in this navigator
+  Route<dynamic> get current => _history.last;
```

Comment:
This should probably return null if _history is empty (currently this would throw).

Reply:
Done.